### PR TITLE
Add opensuse_repos test suite poo#160340

### PR DESCRIPTION
**Pull request by lkocman**
Add basic coverage of openSUSE-repos 

Hello this is my first test, so any help is welcome :-)

- Related ticket: https://github.com/openSUSE/openSUSE-repos/issues/54
- Related discussion: https://lists.opensuse.org/archives/list/factory@lists.opensuse.org/thread/DP3CC5RICD2GJVYBC7OZXEQUBG2Q3G4J/
- Verification run: (I don't have my private instance)

### CODE REVIEW

**Comment by lkocman** [comment]
@bmwiedemann @sysrich how can we make sure that we also runt the check on MicroOS/Slowroll?

**Comment by foursixnine** [comment]
In general looks good, I'll try to schedule the VR run in a bit

File: tests/console/opensuse_repos.pm [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+use strict;
+use warnings;
+use testapi;
+use utils;
+use Utils::Architectures;
+use version_utils qw(is_tumbleweed is_leap is_leap_micro is_microos is_slowroll);
+use serial_terminal 'select_serial_terminal';
+
+sub run {
+    my $timeout = 300;
```
Comment (grisu48) [inline tests/console/opensuse_repos.pm:25]: Unused variable?

File: tests/console/opensuse_repos.pm [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+# SPDX-License-Identifier: FSFAP
+
+# Package: openSUSE-repos
+# Summary: Basic checks for openSUSE-repos.
+
+# - Install openSUSE-repos
+# - Innstall openSUSE-repos-NVIDIA on aarch64 and x86_64
+# - Import gpg keys and refresh repositories
+# - Flavors of openSUSE-repos packages correspond with flavor of openSUSE-release
+# Maintainer: Lubos Kocmman <lubos.kocman@suse.com>
```
Comment (grisu48) [inline tests/console/opensuse_repos.pm:13]: Feel free to add me as maintainer as well if you want to share the burden :wink:

**Review by grisu48 — COMMENTED** [review]
Two suggestions, otherwise LGTM.

File: tests/console/opensuse_repos.pm (hunk 1 above)
Comment (lkocman) [reply tests/console/opensuse_repos.pm:25]: Ah good point, this was originally carried over from uninstall. Will drop.

File: tests/console/opensuse_repos.pm (hunk 2 above)
Comment (lkocman) [reply tests/console/opensuse_repos.pm:13]: will happily do!

**Comment by lkocman** [comment]
@grisu48 all set!. Thank you @foursixnine

**Review by grisu48 — APPROVED** [review]
LGTM. Let's wait for the VRs

**Comment by foursixnine** [comment]
- opensuse-15.6-DVD-x86_64-Build696.2-extra_tests_textmode@64bit -> [](https://openqa.opensuse.org/tests/4200020/badge)
- opensuse-Tumbleweed-DVD-x86_64-Build20240515-extra_tests_textmode@64bit -> [](https://openqa.opensuse.org/tests/4200013/badge)

@lkocman

**Comment by lkocman** [comment]
Thank you @foursixnine !

**Review by foursixnine — APPROVED** [review]

**Comment by grisu48** [comment]
VRs passing: https://openqa.opensuse.org/tests/4200020#step/opensuse_repos/15 and https://openqa.opensuse.org/tests/4200013#step/opensuse_repos/15

**Pull request merged**